### ISSUE TITLE
Site concept

### ISSUE THREAD

**Issue by TorinAsakura** [torin-asakura/web#32]
https://disk.yandex.ru/i/z5Uc4L0koVtjuw - видео для понимания процесса

https://www.figma.com/file/n4uFtQZBib1xCC4Ea9s0AN/Torin-Asakura-web-site - диз

Основной посыл сайта - аудиовизуал.
То есть - каждое действие должно сопровождаться звуковым отзывом, это есть на видео.

Основная фича - это сцена по которой можно передвигаться за счёт мыши в рамках координатной сетки, то есть, не только вверх/вниз, но и по диагонали, так как сетка тут не классическая XY (при условии, что там есть объекты)

Переходы между сущностями (страницы/записи) должно происходить без перезагрузок, а фон должен плавно меняться.

Также на сайте предполагается фоновая музыка, это важный аспект.

**Comment by ArtemHovsky** [comment]
А в чем здесь задача?

**Comment by TorinAsakura** [comment]
@ArtemHovsky Доделать страницы. Состояние. Переходы.

**Comment by ArtemHovsky** [comment]
Переделать стартовый экран с возможность выбора входа на сайт (звук вкл/выкл)
Доработать стиль, внедрить фирменные цвета (фиолетово-синий), показать варианты отображения проектов, услуг, блога в общем экране. 

Утвердить стилистику концепта на нескольких экранах.

**Comment by ArtemHovsky** [comment]
@TorinAsakura 
Посмотри пожалуйста вариант концепции экранов
https://www.figma.com/file/n4uFtQZBib1xCC4Ea9s0AN/Torin-Asakura-web-site?node-id=1446%3A3255&t=gJ61PUh5JAdfcsnJ-1

**Comment by TorinAsakura** [comment]
@ArtemHovsky А куда смотреть-то? На переключение со звуком/без? И что там бирюзовый гексагон образовался с текстом внутри? Что это?

**Comment by ArtemHovsky** [comment]
Да. Как договаривались, с начало определимся со стилистикой и цветами в пределах главного экрана и потом будем двигаться дальше
